### ISSUE TITLE
bootstrap script installs transitional package git-core

### ISSUE THREAD

**Issue by rfc1036** [letsencrypt/letsencrypt#1179]
The correct package for Debian systems is "git".

**Comment by kuba** [comment]
`git-core` is still present and on some versions of Debian it provides much smaller package than the whole `git` suite.

@hlieberman, @fmarier, can you provide input on this, please?

**Comment by fmarier** [comment]
`git-core` is an dummy package whose only purpose is to install the `git` package:

```
$ apt show git-core
Package: git-core
Source: git
Version: 1:2.1.4-2.1
Installed-Size: 1,024 B
Maintainer: Gerrit Pape <pape@smarden.org>
Depends: git (>> 1:1.7.0.2)
Homepage: http://git-scm.com/
Tag: role::dummy
Section: vcs
Priority: optional
Download-Size: 1,494 B
APT-Sources: http://httpredir.debian.org/debian/ jessie/main amd64 Packages
Description: fast, scalable, distributed revision control system (obsolete)
```

**Comment by kuba** [comment]
Okay, it's even the case in squeeze: https://packages.debian.org/squeeze/git-core: "This is a transitional dummy package. The 'git-core' package has been renamed to 'git', which has been installed automatically. This git-core package is now obsolete, and can safely be removed from the system."

Would you like to submit simple fix, @rfc1036?

**Comment by xtaran** [comment]
Fix is in #1186.
